hawc/main/urls.py

```diff
@@ -3,8 +3,35 @@
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.views.generic import TemplateView
+from rest_framework import permissions
+from rest_framework.schemas import get_schema_view
 
+from hawc.apps.animal.urls import router as animal_api
 from hawc.apps.assessment import views
+from hawc.apps.assessment.urls import router as assessment_api
+from hawc.apps.bmd.urls import router as bmd_api
+from hawc.apps.epi.urls import router as epi_api
+from hawc.apps.epimeta.urls import router as epimeta_api
+from hawc.apps.invitro.urls import router as invitro_api
+from hawc.apps.lit.urls import router as lit_api
+from hawc.apps.mgmt.urls import router as mgmt_api
+from hawc.apps.riskofbias.urls import router as riskofbias_api
+from hawc.apps.study.urls import router as study_api
+from hawc.apps.summary.urls import router as summary_api
+
+open_api_patterns = [
```

Comment:
All of the modular url dispatchers used a router object for their API endpoints, so I just imported them all to the main url dispatcher for the OpenAPI to traverse.

Reply:
The include in the urls above can use a string, eg., "hawc.apps.animal.urls"; can these includes use the str version so we don't have to have the imports

Reply:
I tried it with a string parameter, [but it turns out that can only be done with modules.](https://docs.djangoproject.com/en/3.0/ref/urls/#include)

I did clean up the import a bit, and just used full paths within the code instead of creating a new namespace for all the imports. I think the tradeoff looks a little cleaner this way, let me know what you think.